src/main/java/org/circle8/dao/PuntoReciclajeDao.java

```diff
@@ -33,6 +33,7 @@ public class PuntoReciclajeDao extends Dao{
 		LEFT JOIN "TipoResiduo" AS tr on tr."ID" = prtr."TipoResiduoId"
 		LEFT JOIN "Ciudadano" AS ciu on pr."CiudadanoId" = ciu."ID"
 		    WHERE 1=1
+		    AND pr."CiudadanoId" IS NOT NULL
```

Comment:
Si la unica diferencia con `PuntoVerde` es esta, no deberia tener un Service aparte el `PuntoVerde`, y se deberia usar el mismo list que para `PuntoReciclaje`. Y que con un boolean o algo por el estilo agregue el `IS NOT NULL` o el `IS NULL`